x/mongo/driver/topology/server_test.go

```diff
@@ -1028,19 +1265,16 @@ type processErrorTestConn struct {
 	// Embed a driver.Connection to quickly implement the interface without
 	// implementing all methods.
 	driver.Connection
-	stale bool
-	tv    *description.TopologyVersion
+	description description.Server
+	stale       bool
 }
 
-func newProcessErrorTestConn(tv *description.TopologyVersion) *processErrorTestConn {
+func newProcessErrorTestConn(wireVersion *description.VersionRange, stale bool) *processErrorTestConn {
```

Comment:
Can we put this and "newServerDescription" in the namespace? i.e. either both as package test functions, or both as closure functions in the test function?

Reply:
The `processErrorTestConn` can't be defined in the `ProcessError` test function because it needs to define methods to satisfy the `driver.Connection` interface. However, I can move `newServerDescription` to the package scope.